scout/index.md

```diff
@@ -51,6 +51,10 @@ _The following video shows how to enable Docker Scout on your repositories_.
 
 ### Docker Desktop
 
+> **Note**
+>
+> There is a 3gb size limit on images analyzed by Docker Scout in Docker Desktop.
```

Comment:
```suggestion
> There is a 3GB size limit on images analyzed by Docker Scout in Docker Desktop.
```

Another thing for the Style Guide?

Reply:
@aevesdocker Actually, it's already there, I wonder why the vale rule didn't pick it up, I will take a look…